### PR TITLE
index postsubmit job by branch

### PR DESCRIPTION
**Pull request by rkpagadala**
index postsubmit job by branch so that we can look for correct green sha based on the tests required for  a particular branch

### CODE REVIEW

**Comment by sebastienvas** [comment]
https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L263

**Comment by rkpagadala** [comment]
@chxchx please follow this up by reading from config map

File: toolbox/githubctl/main.go [hunk 1]
```diff
@@ -120,6 +138,7 @@ func preprocessProwResults() map[string]map[string]bool {
 			}
 		}()
 	}
+	postSubmitJobs := postSubmitJobsMap[*baseBranch]
```
Comment (sebastienvas) [inline toolbox/githubctl/main.go:141]: you mgith want check that the key exist, and fail otherwise

Comment (rkpagadala) [reply toolbox/githubctl/main.go:141]: thanks seb. fixed it.

**Pull request merged**